crates/sui-indexer/src/indexer_reader.rs

```diff
@@ -997,11 +997,10 @@ impl IndexerReader {
             format!(
                 "( \
                     SELECT *
-                    FROM tx_senders s
+                    FROM event_senders s
```

Comment:
there should only be a single sender for the events of a tx right? or can this change in the future. I think we can save some storage and possibly improve query performance if we join on just tx_sequence_number against a table that tracks just tx <-> sender

Reply:
Yeah, I did have this thought (whether we could get rid of `event_senders` as well). The only thing that gave me pause was that in GraphQL, we then lose the property that every filter is implemented by an index-only scan on a table that will return a sorted sequence of keys, and the join between the `tx <-> sender` table and the event sequence number table would be a potential place where the query planner could mess up.